docs/FAQ.md

```diff
@@ -111,6 +111,79 @@ store arbitrary files in `<your-git-repo>/scratch/`.
 
 You can find more details on `gitignore` files [here][gitignore].
 
+### How can I avoid committing my local-only changes to tracked files?
```

Comment:
I wonder if the "Can I prevent Jujutsu from recording my unfinished work? I'm not ready to commit it." question should be combined with this one or at least link to this one.

Reply:
I added a link in that question, thanks